sdk/communication/AzureCommunicationChat/Swagger/swagger.json

```diff
@@ -0,0 +1,1711 @@
+{
```

Comment:
We do not need this file here any more, right? I assume we are referencing the file in the spec repo.

Reply:
Correct - removed, thanks